### PR TITLE
add markAsCopy and dynamicPTable to config.md

### PR DESCRIPTION
**Pull request by Tastaturberuf**

### CODE REVIEW

**Comment by Tastaturberuf** [comment]
i have the table rearanged. thats why the whitespace changes in more then two lines

**Comment by fritzmg** [comment]
`markAsCopy` is a new feature and not present in Contao 4.4. Can you check in which Contao version this was added and add a note somewhere?

**Review by fritzmg — CHANGES_REQUESTED** [review]
`markAsCopy` is a new feature and not present in Contao 4.4. Can you check in which Contao version this was added and add a note somewhere?

**Comment by Tastaturberuf** [comment]
oh crap, yes this is 4.8. i'll fix thats

**Comment by Tastaturberuf** [comment]
Source: https://github.com/contao/core-bundle/commit/e45c20cc5c5b642ae0b17bc413275efc9836062e

File: docs/dev/reference/dca/config.md [hunk 1]
```diff
@@ -33,23 +33,25 @@ $GLOBALS['TL_DCA']['tl_example']['config'] = [
...
-| switchToEdit       | true/false (`bool`)            | Activates the "save and edit" button when a new record is added (sorting mode 4 only).                                                          |
-| enableVersioning   | true/false (`bool`)            | If true, Contao saves the old version of a record when a new version is created.                                                                |
-| doNotCopyRecords   | true/false (`bool`)            | If true, Contao will not duplicate records of the current table when a record of its parent table is duplicated.                                |
-| doNotDeleteRecords | true/false (`bool`)            | If true, Contao will not delete records of the current table when a record of its parent table is deleted.                                      |
+| markAsCopy         | Field name (`string`)             | Mark the new record with "copy of" in this field name. Since Contao 4.6                                                                         |
```
Comment (fritzmg) [inline docs/dev/reference/dca/config.md]:
```suggestion
| markAsCopy         | Field name (`string`)             | Appends "(copy)" to this field when copying a record (Contao __4.6__ and up).                                                                         |
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/dev/reference/dca/config.md (hunk 1 above)
Comment (fritzmg) [reply docs/dev/reference/dca/config.md]: Ich kann leider nicht direkt in deinen branch comitten, müsstest du selbst comitten ;)

**Review by fritzmg — CHANGES_REQUESTED** [review]

**Review by fritzmg — APPROVED** [review]

**Comment by fritzmg** [comment]
Thank you @Tastaturberuf :)

**Pull request merged**